gradle/dependency-versions.gradle

```diff
@@ -17,4 +17,5 @@ ext {
     commonsVersion = "1.5"
     bouncycastleVersion = "1.52"
     javaxVersion = "3.0.1"
+    helixVersion = "0.6.7-SNAPSHOT"
```

Comment:
Will have to figure something out for the general case. For now, Helix is planning to cut a release later this week, so we are good.